brian_cheung/conditionals.md

```diff
@@ -8,14 +8,24 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 ```ruby
 color = "Magenta"
 
-# write your code here
+if color == "Magenta"
+	puts "Magenta is my second favorite color"
+end
 ```
 
 Now, turn your color conditional into a method called `color_checker` that takes any color as an argument.
 
 ```ruby
 def color_checker(color)
-  # write your code here
+	if color == "Chartreuse"
+		puts "#{color} is my favorite color"
+	elsif color == "Magenta"
+		puts "#{color} is my second favorite color"
+	elsif color == "Periwinkle"
+		puts "#{color} is my third favorite color"
+	else
+		puts "I hate the color #{color}"
+	end
 end
 ```
 
```

Comment:
Looks good! The spacing is a little off in GitHub, so check your Sublime settings - if you indent using spacing and have a tab width of 2, it should look normal in GitHub.